v8/payloads/auditLog.ts

```diff
@@ -19,22 +19,72 @@ import type { APIWebhook } from './webhook';
  * https://discord.com/developers/docs/resources/audit-log#audit-log-object-audit-log-structure
  */
 export interface APIAuditLog {
+	/**
+	 * Webhooks found in the audit log
+	 *
+	 * See https://discord.com/developers/docs/resources/webhook#webhook-object
```

Comment:
This should use `@see`, now that TypeScript 4.1 [added support for it](https://devblogs.microsoft.com/typescript/announcing-typescript-4-1/#jsdoc-see-tag), same for everything else.

Reply:
Intentionally chose against it so that the link is actually clickable in VSC.